Project.toml

```diff
@@ -1,4 +1,25 @@
 name = "FiniteHorizonPOMDPs"
 uuid = "8a13bbfe-798e-11e9-2f1c-eba9ee5ef093"
-authors = ["Zachary Sunberg <sunbergzach@gmail.com>"]
+authors = ["Tomas Omasta <omastto1@fel.cvut.cz> and contributors"]
 version = "0.1.0"
+
+[deps]
+DiscreteValueIteration = "4b033969-44f6-5439-a48b-c11fa3648068"
```

Comment:
Does this need to have DiscreteValueIteration as a dependency?

Reply:
I mistakenly ran tests from the parent environment instead of the testing environment, fixed on my side.